src/test/regress/expected/failure_create_index_concurrently.out

```diff
@@ -0,0 +1,153 @@
+--
+-- failure_create_index_concurrently
+-- test create index concurrently command
+-- failure.
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SET citus.shard_count = 2; -- one per worker
+CREATE TABLE index_test(id int, value_1 int, value_2 int);
+SELECT create_distributed_table('index_test', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- kill the connection when create command is issued
+SELECT citus.mitmproxy('conn.onQuery(query="CREATE").kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT citus.clear_network_traffic();
+ clear_network_traffic 
+-----------------------
+ 
+(1 row)
+
+CREATE INDEX CONCURRENTLY idx_index_test ON index_test(id, value_1);
+ERROR:  CONCURRENTLY-enabled index command failed
+DETAIL:  CONCURRENTLY-enabled index commands can fail partially, leaving behind an INVALID index.
+HINT:  Use DROP INDEX CONCURRENTLY IF EXISTS to remove the invalid index, then retry the original command.
+SELECT citus.dump_network_traffic();
+                                                                                                                                                                                                                                                                       dump_network_traffic                                                                                                                                                                                                                                                                        
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
This is a discussion we've made with @lithp  as well, but we haven't concluded the discussion.

I think this `dump_network_traffic ()` is super useful by itself both during development and debugging. But, do you see necessary to add to regression tests? I'm OK with keeping it, but I don't add them to the PRs that I open because they seem too verbose to add to the output. I even hit few cases where the output isn't consistent such as [here](https://github.com/citusdata/citus/pull/2244/commits/c723121d1d4f4962d2aaeb498635f8b979adab5e#r199416333). (I don't expect inconsistencies for this case, but, just wanted to note)

Reply:
I think you're right that they're too verbose to add to every line. A style that I think makes sense is to show the dump just once at the top, run a happy-case so future readers can see exactly what the rest of the test is mucking with. These two dumps don't really show me what is supposed to happen, they just show the same truncated log twice.

I don't want to impose that style on anyone though if they don't agree with it.